src/taskManager.ts

```diff
@@ -39,16 +40,16 @@ export class TaskManager {
         zoomLevel: parameters.zoomLevel,
         tilesPath: join(this.tilesDirectoryPath, parameters.tilesPath),
         packageName: parameters.packageName,
-        callbackURL: parameters.callbackURL,
+        callbackURLs: parameters.callbackURLs,
```

Comment:
make sure this is what the trigger sends ("callbackURLs')

Reply:
trigger does send `callbackURLs` rather than `callbackURL`